pkgs/phps.nix

```diff
@@ -1,4 +1,6 @@
-nixpkgs:
+{
```

Comment:
These changes probably do not belong into this commit.

Reply:
Fixed!

Reply:
When removing that, I need to change :

```nix
      overlays.default = import ./pkgs/phps.nix {
        inherit nixpkgs;
      };
```

in

```nix
      overlays.default = import ./pkgs/phps.nix nixpkgs.outPath;
```

Should I do it?

Reply:
Honestly, I prefer leaving it as such, WDYT ?

Reply:
Yeah, I would just keep it as it was before. The refactoring is okay but it is not really related to this change so I would prefer to keep it separated.

Reply:
Okidocky.